src/os_esp32.cc

```diff
@@ -607,7 +608,7 @@ class HeapSummaryPage {
       case EVENT_SOURCE_MALLOC_TAG: return "event source";
       case OTHER_THREADS_MALLOC_TAG: return "other threads";
       case THREAD_SPAWN_MALLOC_TAG: return "thread spawn";
-      case NULL_MALLOC_TAG: return "null tag";
+      case NULL_MALLOC_TAG: return "untagged allocations";
```

Comment:
They are all allocations. Maybe just untagged?
```suggestion
      case NULL_MALLOC_TAG: return "untagged";
```